CHANGELOG.md

```diff
@@ -6,11 +6,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## 2.1.0 - 2024-02-8
+## Unreleased
```

Comment:
this shouldn't be here I suppose

Reply:
I just added it as a placeholder for the future